Resources/doc/usage.rst

```diff
@@ -0,0 +1,136 @@
+####################
+Using filter manager
+####################
+
+================
+Using controller
+================
+
+~~~~~~~~~~~~~~~~~~~~~~~~
+Using default controller
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you set up your `*managers* <manager.rst>`_ you don't need to create a controller for each one,
+you can just use default manager controller `ONGRFilterManagerBundle:Manager:manager`.
+
+Example:
+
+.. code-block:: yaml
+
+    #src/Acme/DemoBundle/Resources/config/routing.yml
+    ongr_search_page:
+        pattern: /list
+        methods:  [GET]
+        defaults:
+            _controller: ONGRFilterManagerBundle:Manager:manager
+            managerName: "item_list"
+            template: "AcmeDemoBundle:List:results.html.twig"
+
+..
+
+This specific example will render template `AcmeDemoBundle:List:results.html.twig`,
+with `SearchResponse` from `FiltersManager` named `item_list`.
+
+------------------------
+Using custom controller.
+------------------------
+
+You can still use custom controller by getting your needed manager from the container.
+This way you can add your custom variables if needed.
+
+Example:
+
+.. code-block:: yaml
+
+    #src/Acme/DemoBundle/Resources/config/routing.yml
+    ongr_search_page:
+        pattern: /list
+        methods:  [GET]
+        defaults:
+            _controller: AcmeDemoBundle:List:index
+
+..
+
+.. code-block:: php
+
+    //src/Acme/DemoBundle/Controller/ListController.php
```

Comment:
Yes, it  should probably read something like this:
` // File location: src/.../ListController.php.`